store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/operation/handler/impl/FederatedAggregateHandlerTest.java

```diff
@@ -92,6 +101,14 @@ public void shouldAggregateDuplicatesFromDiffStores() throws Exception {
         final String graphNameB = "b";
 
         final Context context = new Context(new User());
+        Properties properties = PROPERTIES.getProperties();
+        AccumuloProperties propsA = new AccumuloProperties();
+        propsA.setProperties(properties);
+        propsA.setInstance(properties.getProperty(AccumuloProperties.INSTANCE_NAME) + "A");
```

Comment:
Again I presume this is just to be clearer and make debugging easier?

Reply:
It also causes a table to be unique to this.